spec/decorators/user_decorator_spec.rb

```diff
@@ -1,48 +1,92 @@
 require "rails_helper"
 
 RSpec.describe UserDecorator, type: :decorator do
-  let(:user) { build_stubbed(:user) }
+  let_it_be_changeable(:saved_user) { create(:user) }
```

Comment:
I've never seen `let_it_be_changeable` used before--what does it do?

Reply:
Reloads the object for each test, the contributor that sent the PR that added it wrote a beautiful explanation for the reason why: https://github.com/thepracticaldev/dev.to/pull/5407

You can find the code itself in `spec/support/initializers/test_prof.rb`

Reply:
Awesome; thank you!